web/src/TabNav.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from "react"
+import { Link } from "react-router-dom"
+import { SidebarItem } from "./Sidebar"
+import { ResourceView } from "./HUD"
+import "./TabNav.scss"
+type NavProps = {
+  resourceName: string
+  sidebarItems: Array<SidebarItem>
```

Comment:
it seems weird to pass all the sidebar items into TabNav. why should it know about all the sidebar items? maybe just pass in the previewResourceURL?